Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocCheckBox.css

```
@@ -3,38 +3,64 @@
 .bocCheckBox
 {
   margin-right: 0.5em;
+  white-space: nowrap;
+}
+
+.bocCheckBox input
+{
+  position: absolute;
+  z-index: -1;
+}
+
+.bocCheckBox img
+{
+  margin: 2px;
   vertical-align: middle;
+  height: 16px;
+  width: 16px;
+}
+
+.bocCheckBox input + span::before
+{
+  margin: 2px;
+  visibility: visible;
+  background-image: url('../Image/sprite.svg#CheckBoxFalse');
+  width: 16px;
+  height: 16px;
   display: inline-block;
-  white-space: nowrap;
+  content: "";
+  z-index: 1;
+  position: relative;
 }
 
-.bocCheckBox.readOnly
+.bocCheckBox input[checked] + span::before
 {
+  background-image: url('../Image/sprite.svg#CheckBoxTrue');
 }
 
-.bocCheckBox.disabled
+.bocCheckBox,
+.bocCheckBox > input + span
 {
+  display: flex;
+  align-items: center;
 }
 
-.bocCheckBox input
+/* Fixes for element focus */
+
+.bocCheckBox span[role=checkbox]:focus
 {
-  margin-left: 4px;
-  margin-right: 3px;
-  padding: 0px;
-  vertical-align: -0.1em;
+  outline: none;
 }
 
-.bocCheckBox span + span
+.blink .bocCheckBox span[role=checkbox]:focus > img,
+.blink .bocCheckBox input:focus + span::before
 {
-  vertical-align: middle;
+  box-shadow: 0 0 0 1px black;
```

```suggestion
  box-shadow: 0 0 0 2px black;
  border-radius: var(--remotion-themed-border-radius);
```

We should consider creating properties for the blink and mozilla box shadows in common.css. Question: do this now?